lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -160,6 +162,40 @@ void testMain() {
       expect(ui.PlatformDispatcher.instance.textScaleFactor, findBrowserTextScaleFactor());
     });
   });
+
+  test('A ViewConfiguration asserts that both window and view are not provided', () {
+    expect(() {
+      // ignore: deprecated_member_use
+      final EngineFlutterWindow window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
+      ui.ViewConfiguration(
+        window: window,
+        view: window,
+      );
+    }, throwsAssertionError);
+  });
+
+  test("A ViewConfiguration's view and window are backed with the same property", () {
+    final ui.FlutterView window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
+    final ui.ViewConfiguration viewConfiguration = ui.ViewConfiguration(view: window);
+    // ignore: deprecated_member_use
+    expect(viewConfiguration.window, window);
+    // ignore: deprecated_member_use
+    expect(viewConfiguration.window, viewConfiguration.view);
+  });
+
+  test('Initialize a ViewConfiguration with a window', () {
+    final ui.FlutterView window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
+    // ignore: deprecated_member_use
+    ui.ViewConfiguration(window: window);
```

Comment:
This needs an expectation too:

```suggestion
    ui.ViewConfiguration(window: window);
    expect(viewConfiguration.view, window);
```